web/src/components/SideBar.tsx

```diff
@@ -22,6 +23,16 @@ interface DefaultsProps {
 
 type Props = ConnectedProps & DispatchProps & DefaultsProps & RouteComponentProps<{service: string}>
 
+const sidebarItemClass = (error: AggregatedError): string => {
+  if (error.severity === "info") {
+    return "sidebar-item-info"
+  } else if (error.severity === "warning") {
+    return "sidebar-item-warning"
+  } else {
+    return "sidebar-item-error"
```

Comment:
Maybe in the future if we tend to treat severity this way, we can think about introducing a client-side normalized representation for errors where `severity` defaults to `error` and is not optional anymore